src/triagebot/transfer.md

```diff
@@ -0,0 +1,36 @@
+# Issue Transfer
+
+The `transfer` command allows you to transfer a GitHub issue from one repository to another.
+
+## Usage
+
+To transfer an issue to another repository, enter a comment with the form:
+
+`@rustbot transfer` *repository-name*
+
+It is recommended to also include a comment explaining why you are transferring. For example:
+
+```
+Transferring to rust-lang/cargo since this is an issue with how cargo
+implements diagnostic reports.
+
+@rustbot transfer cargo
+```
+
+**IMPORTANT: There will be no visual indication that the issue is being transferred.** Due to GitHub API limitations, you will not see any activity. **You must reload the page** to view the issue in its new location. It may take a few moments for GitHub to transfer all the data.
+
+**WARNING:** Transferring is a partially destructive command. For example, labels and milestones that don't exist in the target repository will be removed.
```

Comment:
```suggestion
**WARNING:** Transferring is a partially destructive command. For example, labels and milestones that don't exist in the target repository will be removed from the issue.
```

clarification, might not be necessary. At the moment, this sentence could be misunderstood to remove the labels and milestones *from the repository*. Arguably that's a stretch. However, people might get thrown off by the words *WARNING* and *destructive* and start doubting themselves, dunno >.<